oauth2client/django_util/decorators.py

```diff
@@ -0,0 +1,50 @@
+from django.shortcuts import redirect
+from oauth2client.django_util.storage import get_storage
+from six import wraps
+from . import settings
+
+
+def _build_scope_query_param(scopes):
```

Comment:
Does django not have a built-in function for this? Mabye [`reverse`](https://docs.djangoproject.com/en/dev/ref/urlresolvers/#reverse)?